TGC/common/buildings.txt

```diff
@@ -732,15 +732,15 @@
 		# Audax Validator "." Ignore_NEXT
 		type = university
 		# Audax Validator "." Ignore_NEXT
-		cost = 30000
+		cost = 15000
 		goods_cost =
 		{
-			steel = 20
+			steel = 30
 			lumber = 90
 			cement = 90
-			paper = 10
-			dye = 10
-			glass = 10
+			paper = 20
+			#dye = 10
```

Comment:
These ones too

Reply:
Removed due UI ugliness, it cant fit more than 4 resources properly